scheduler/reconcile.go

```diff
@@ -145,8 +145,8 @@ type delayedRescheduleInfo struct {
 }
 
 func (r *reconcileResults) GoString() string {
-	base := fmt.Sprintf("Total changes: (place %d) (destructive %d) (inplace %d) (stop %d)",
-		len(r.place), len(r.destructiveUpdate), len(r.inplaceUpdate), len(r.stop))
+	base := fmt.Sprintf("Total changes: (place %d) (destructive %d) (inplace %d) (stop %d) (disconnect %d) (reconnect %d)",
+		len(r.place), len(r.destructiveUpdate), len(r.inplaceUpdate), len(r.stop), len(r.disconnectUpdates), len(r.reconnectUpdates))
```

Comment:
Possibly nitpicky: because this report is for "Total changes", is calling the change `disconnect` and `reconnect` the right name? That reads to me like the scheduler is asking for the allocs to be disconnected or reconnected. (Might be a good question to throw to the team for bikeshedding 😀 )